openc3/lib/openc3/api/cmd_api.rb

```diff
@@ -99,6 +99,7 @@ def cmd_raw_no_checks(*args, **kwargs)
     # @param interface_name [String] The interface to send the raw binary
     # @param data [String] The raw binary data
     def send_raw(interface_name, data, scope: $openc3_scope, token: $openc3_token)
+      interface_name.upcase!
```

Comment:
This is potentially modifying the callers string unexpectedly.  Probably should use interface_name = interface_name.upcase, here and other places where the value you are using upcase! is an argument.